modules/creation/good-practices.md

```diff
@@ -85,9 +85,9 @@ add a preliminary check before using them (I.e with `extension_loaded`). This pr
 
 **A few recommendations for your email templates**
 
-- Use our [official SDK](https://github.com/PrestaShop/email-templates-sdk) to develop your emails: 
+- Use our [official SDK](https://github.com/PrestaShopCorp/email-templates-sdk) to develop your emails: 
```

Comment:
@thomasnares note for the future, you can create an issue because we need to rework this paragraph, it is about PrestaShop corp and Addons Marketplace